assertj-core/src/test/java/org/assertj/core/api/keystore/KeyStoreAssert_containsAlias_Test.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2023 the original author or authors.
+ */
+package org.assertj.core.api.keystore;
+
+import org.junit.jupiter.api.Test;
+
+import java.security.KeyStore;
+import java.security.KeyStoreSpi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.BDDAssertions.then;
+import org.assertj.core.test.KeyStores;
+import static org.assertj.core.error.KeyStoreShouldContainAlias.shouldContainAlias;
+import static org.assertj.core.util.AssertionsUtil.expectAssertionError;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class KeyStoreAssert_containsAlias_Test {
```

Comment:
minor: test classes don't need to be `public`, let's make it package private (no keyword)

Reply:
Updated